### DISCUSSION TITLE
请问有类似Three.js的AxesHelper的坐标轴辅助工具么

### DISCUSSION THREAD

**Discussion by zhhbinn** [galacean/discussion#16] [Engine Toolkit] [answered]
对于透视投影感觉会帮助性很大

**Comment by JujieX** [comment, marked as answer]
> 对于透视投影感觉会帮助性很大

目前和这个工具类似的是
1. toolkit 中的 infinite grid #https://github.com/oasis-engine/engine-toolkit/pull/54 是标识了XZ方向的无边网格，目前正在PR阶段
2. toolkit中的 gizmo https://github.com/oasis-engine/engine-toolkit/tree/main/packages/gizmo 可视化拖拽、旋转、缩放，这一版和编辑器耦合较多目前正在重构，最迟预计下个月底可用
3. toolkit 规划的 navigation gizmo #https://github.com/oasis-engine/engine-toolkit/issues/52 标识场景相机变化，切换视图，预计本里程碑完成开发

**Comment by singlecoder** [comment]
你是想做场景搭建用到么
